src/Symfony/Component/VarDumper/Caster/SymfonyCaster.php

```diff
@@ -66,4 +68,30 @@ public static function castHttpClientResponse($response, array $a, Stub $stub, b
 
         return $a;
     }
+
+    public static function castUuid(Uuid $uuid, array $a, Stub $stub, bool $isNested)
+    {
+        $a[Caster::PREFIX_VIRTUAL.'base 58'] = $uuid->toBase58();
```

Comment:
```suggestion
        $a[Caster::PREFIX_VIRTUAL.'base58'] = $uuid->toBase58();
```

What about removing the space? Same for the next line?

Reply:
I agree with those suggestions. I did even think about using the name of the methods:
`toBase58`, etc